examples/fabric/kfold_cv/README.md

```diff
@@ -0,0 +1,45 @@
+## K-Fold Cross Validation
+
+This is an example of performing K-Fold cross validation supported with [Lightning Fabric](https://pytorch-lightning.readthedocs.io/en/stable/fabric/fabric.html). To know more about cross validation, check out [this article](https://sebastianraschka.com/blog/2016/model-evaluation-selection-part3.html#introduction-to-k-fold-cross-validation).
+
+We use the MNIST dataset to train a simple CNN model. We create the k-fold cross validation splits using the `ModelSelection.KFold` [class](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.KFold.html) in the `scikit-learn` library. Ensure that you have the `scikit-learn` library installed.
+
+### Run
+
+Tip: You can easily inspect the difference between the two files with:
+
+```bash
+sdiff train_torch.py train_fabric.py
+```
```

Comment:
I see that you followed the previous examples with fabric and equivalent torch code. The motivation there was to help people who learn about fabric understand the differences between torch and fabric, and to basically show that there are very few of them!

I think when we get to examples like k-fold validation etc. It becomes a bit redundant to have the torch code also. I suggest to drop train_torch.py completely here. This will also be easier to maintain. No strong opinion here, I just think it's cleaner :)

Reply:
I agree it does not make much sense to add it in this folder. I thought there was some test that calls `train_torch.py` from this folder, so for consistency, I added it. I have removed it and updated README accordingly.